icons/icon.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<svg viewBox="0 0 306 306" style="background-color: #4a4a4a" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">
+  <path fill="#ffffff" transform="translate(64, 32)" d="m62.371094,79.371094l29.880005,0q7.369995,-31.25 16.869995,-46.5q9.5,-15.25 24.75,-24.060001q15.25,-8.810001 28.75,-8.810001q7.630005,0 11.5,3q3.880005,3 3.880005,7.120003q0,3.75 -2.630005,6.25q-2.619995,2.5 -6.869995,2.5q-4.380005,0 -16.5,-2.619999q-6,-1.25 -9.380005,-1.25q-8.25,0 -14.869995,10.18q-6.630005,10.189999 -10.630005,27.569996l-6.25,26.620003l33,0l-2,8.879997l-33.369995,0l-19.75,79.750008q-9.130005,36.369995 -29.190002,55.119995q-20.059998,18.75 -43.559998,18.75q-7.880005,0 -11.940002,-3.059998q-4.059998,-3.059998 -4.059998,-7.559998q0,-3.880005 2.809998,-6.190002q2.809998,-2.309998 8.059998,-2.309998q2.630005,0 7.75,0.619995q8.75,1.130005 16.75,1.130005q12.380005,0 18,-5.820007q5.630005,-5.809998 11.25,-28.429993l25.380005,-102.000008l-29.880005,0l2.25,-8.879997z"/>
+</svg>
```

Comment:
So I added an SVG version of the icon so we could use it in the future. The problem is, Firefox doesn't document its support for SVG extension icons, and you have to do it like this in the manifest:
```json
"icons": {
    "16": "icons/icon.svg",
    "32": "icons/icon.svg",
    "48": "icons/icon.svg",
    "96": "icons/icon.svg",
    "128": "icons/icon.svg",
    "256": "icons/icon.svg"
}
```
which sort of sucks.
Chrome [doesn't support SVG extension icons at all](https://bugs.chromium.org/p/chromium/issues/detail?id=29683).